plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcWriteConfig.java

```diff
@@ -22,8 +22,10 @@
 public class JdbcWriteConfig
 {
     public static final int MAX_ALLOWED_WRITE_BATCH_SIZE = 10_000_000;
+    static final int DEFAULT_WRITE_PARALELLISM = 8;
```

Comment:
by default `QueryManagerConfig` defines `maxWriterTasksCount = 100` so this is a drastic reduction of writers. Is it by intention? 
I see, that it's only for jdbc, so it's probably acceptable and 8 is even better than 100.
Should it be documented? 
@jhlodin